web/api/web_api_v1.c

```diff
@@ -776,16 +776,46 @@ static inline void web_client_api_request_v1_info_summary_alarm_statuses(RRDHOST
 static inline void web_client_api_request_v1_info_mirrored_hosts(BUFFER *wb) {
     RRDHOST *rc;
     int count = 0;
+
+    buffer_strcat(wb, "\t\"mirrored_hosts\": [\n");
     rrd_rdlock();
     rrdhost_foreach_read(rc) {
         if (rrdhost_flag_check(rc, RRDHOST_FLAG_ARCHIVED))
             continue;
-        if(count > 0) buffer_strcat(wb, ",\n");
+        if (count > 0)
+            buffer_strcat(wb, ",\n");
+
         buffer_sprintf(wb, "\t\t\"%s\"", rc->hostname);
         count++;
     }
-    buffer_strcat(wb, "\n");
+
+    buffer_strcat(wb, "\n\t],\n\t\"mirrored_hosts_status\": [\n");
+    count = 0;
+    rrdhost_foreach_read(rc)
```

Comment:
This should not be called `rc` it should be called `host`. The part that you copied it from is wrong, the contributor was working on alarms so they reused the iterator name for alarm entries.

Reply:
will change the variable name for clarity